test/dleq.test.ts

```diff
@@ -4,11 +4,11 @@
 // at https://opensource.org/licenses/BSD-3-Clause
 
 import { DLEQParams, DLEQProof, DLEQProver, Elt, Oprf, Scalar } from '../src/index.js'
-
+import './setGroup.js'
```

Comment:
Let's use `describe.each([groupNoble, groupSjcl])` to run both versions using jest.
So no need of ENV variables.

Reply:
```
import { GroupConsNoble } from '../src/groupNoble.js'
import { GroupConsSjcl } from '../src/groupSjcl.js'
import { GroupCons, Oprf } from '../src/index.js'

const groupConsMatch = process.env.GROUP_CONS

export const testGroups = (
    [
        ['noble', GroupConsNoble],
        ['sjcl', GroupConsSjcl]
    ] as const
).filter(([name]) => {
    return !groupConsMatch || name === groupConsMatch
})

export function describeGroupTests(declare: (group: GroupCons) => void) {
    describe.each(testGroups)(`Group-%s`, (_groupName, groupCons) => {
        Oprf.Group = groupCons
        declare(groupCons)
    })
}
```

I believe the above (or something close) ought to keep us both happy :) !